app/auth-portal/src/pages/WorkspaceAuthTokensPage.vue

```diff
@@ -175,8 +199,14 @@ const createAuthToken = useAsyncState(
 /** Name of token to create */
 const createAuthTokenName = ref("");
 
+/** Token modal */
+const tokenDisplayModalRef = ref<InstanceType<typeof Modal> | null>(null);
```

Comment:
```suggestion
const tokenDisplayModalRef = ref<InstanceType<typeof Modal>>();
```

Other places in the codebase do this too. It's DEFINITELY just a nit.